### PR TITLE
Add accepted revisions for SDL-0224 - Navigation Subscription Buttons

### PR DESCRIPTION
**Pull request by joeljfischer**
### Changes:
* Add additional button names.
* Specify that navigation services now support the `ButtonPress` RPC and the new button names.

### CODE REVIEW

**Comment by joeljfischer** [comment]
@theresalech this is ready for review. Thanks!

File: proposals/0224-navigation-subscription-buttons.md [hunk 1]
```diff
@@ -29,9 +29,19 @@ The proposed solution is to add new subscription buttons specifically for these
     <element name="NAV_PAN_DOWN_LEFT" />
     <element name="NAV_PAN_LEFT" />
     <element name="NAV_PAN_UP_LEFT" />
+    <element name="NAV_TILT_TOGGLE">
+        <description>If supported, this toggles between a top-down view and an angled/3D view. If you app supports different, but substatially similar options, then you may implement those. If you don't implement these or similar options, do not subscribe to this button.</description>
```
Comment (theresalech) [inline proposals/0224-navigation-subscription-buttons.md]:
```suggestion
        <description>If supported, this toggles between a top-down view and an angled/3D view. If your app supports different, but substantially similar options, then you may implement those. If you don't implement these or similar options, do not subscribe to this button.</description>
```

File: proposals/0224-navigation-subscription-buttons.md [hunk 2]
```diff
@@ -29,9 +29,19 @@ The proposed solution is to add new subscription buttons specifically for these
...
+    </element>
+    <element name="NAV_ROTATE_CLOCKWISE" />
+    <element name="NAV_ROTATE_COUNTERCLOCKWISE" />
+    <element name="NAV_HEADING_TOGGLE">
+        <description>If supported, this toggles between locking the orientation to north or to the vehicle's heading. If you app supports different, but substatially similar options, then you may implement those. If you don't implement these or similar options, do not subscribe to this button.
```
Comment (theresalech) [inline proposals/0224-navigation-subscription-buttons.md]:
```suggestion
        <description>If supported, this toggles between locking the orientation to north or to the vehicle's heading. If your app supports different, but substantially similar options, then you may implement those. If you don't implement these or similar options, do not subscribe to this button.
```

**Comment by joeljfischer** [comment]
@theresalech thanks for the fixes, they are all merged in!

**Pull request merged**